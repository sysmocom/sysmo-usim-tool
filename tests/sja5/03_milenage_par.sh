#!/bin/sh
. ./test-data

# Write and reread with test data
C1="1234567890ABCDEF1234567890ABCDEF"
C2="F1234567890ABCDEF1234567890ABCDE"
C3="EF1234567890ABCDEF1234567890ABCD"
C4="DEF1234567890ABCDEF1234567890ABC"
C5="CDEF1234567890ABCDEF1234567890AB"
R12345="AA:BB:CC:DD:EE"
PARAMS=$C1$C2$C3$C4$C5$R12345
$TOOL -a $ADMPIN -L $PARAMS
$TOOL -a $ADMPIN -l

# Write and reread with factory defaults
C1="00000000000000000000000000000000"
C2="00000000000000000000000000000001"
C3="00000000000000000000000000000002"
C4="00000000000000000000000000000004"
C5="00000000000000000000000000000008"
R12345="40:00:20:40:60"
PARAMS=$C1$C2$C3$C4$C5$R12345
$TOOL -a $ADMPIN -L $PARAMS
$TOOL -a $ADMPIN -l
