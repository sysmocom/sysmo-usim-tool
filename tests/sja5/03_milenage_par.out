sysmoISIM-SJA5 parameterization tool
Copyright (c) 2023 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 35 75 30 35 02 59 C4
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Milenage parameters...
 * Initalizing...
 * New Milenage Parameters for (EF.MILENAGE_CFG):
   R1: 0xaa
   R2: 0xbb
   R3: 0xcc
   R4: 0xdd
   R5: 0xee
   C1: 1234567890abcdef1234567890abcdef
   C2: f1234567890abcdef1234567890abcde
   C3: ef1234567890abcdef1234567890abcd
   C4: def1234567890abcdef1234567890abc
   C5: cdef1234567890abcdef1234567890ab
 * Programming...

Done!
sysmoISIM-SJA5 parameterization tool
Copyright (c) 2023 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 35 75 30 35 02 59 C4
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Milenage parameters...
 * Initalizing...
 * Reading...
 * Current Milenage Parameters:
   R1: 0xaa
   R2: 0xbb
   R3: 0xcc
   R4: 0xdd
   R5: 0xee
   C1: 1234567890abcdef1234567890abcdef
   C2: f1234567890abcdef1234567890abcde
   C3: ef1234567890abcdef1234567890abcd
   C4: def1234567890abcdef1234567890abc
   C5: cdef1234567890abcdef1234567890ab

Done!
sysmoISIM-SJA5 parameterization tool
Copyright (c) 2023 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 35 75 30 35 02 59 C4
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Milenage parameters...
 * Initalizing...
 * New Milenage Parameters for (EF.MILENAGE_CFG):
   R1: 0x40
   R2: 0x0
   R3: 0x20
   R4: 0x40
   R5: 0x60
   C1: 00000000000000000000000000000000
   C2: 00000000000000000000000000000001
   C3: 00000000000000000000000000000002
   C4: 00000000000000000000000000000004
   C5: 00000000000000000000000000000008
 * Programming...

Done!
sysmoISIM-SJA5 parameterization tool
Copyright (c) 2023 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 35 75 30 35 02 59 C4
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading Milenage parameters...
 * Initalizing...
 * Reading...
 * Current Milenage Parameters:
   R1: 0x40
   R2: 0x0
   R3: 0x20
   R4: 0x40
   R5: 0x60
   C1: 00000000000000000000000000000000
   C2: 00000000000000000000000000000001
   C3: 00000000000000000000000000000002
   C4: 00000000000000000000000000000004
   C5: 00000000000000000000000000000008

Done!
